test/ufuzz.js

```diff
@@ -440,7 +502,7 @@ for (var round = 0; round < num_iterations; round++) {
     var beautify_result = run_code(beautify_code);
 
     try {
-      var uglify_code = minify(beautify_code, {
+      var uglify_code = minify(original_code, {
```

Comment:
Testing beautify on input revealed a couple of bugs. It would be nice if were an option, disabled by default to speed up fuzzing.

When there's a failure can both the minified output code as well as beautified output code be logged? That way it's easier to diagnose. We have to beautify the output anyway.